src/patternfly/components/Select/select-checkbox-groups.hbs

```diff
@@ -0,0 +1,48 @@
+  {{/if}}>
+  {{#> form}}
+    {{#> form-group form-group--modifier="pf-c-form__fieldset" form-group--attribute=(concat 'aria-labelledby="' id '-label"')}}
+      <div class="pf-c-select__menu-group">
+        <div class="pf-c-select__menu-group-title">Status</div>
+        {{#> check  check--type="label" check--attribute=(concat 'for="' id '-running"') check--modifier="pf-c-select__menu-item"}}
+          {{#> check-input check-input--attribute=(concat 'type="checkbox" id="' id '-running" name="running"')}}{{/check-input}}
+          {{#> check-label check-label--type="span"}}Running{{/check-label}}
+        {{/check}}
+        {{#> check  check--type="label" check--attribute=(concat 'for="' id '-stopped"') check--modifier="pf-c-select__menu-item"}}
+          {{#> check-input check-input--attribute=(concat 'type="checkbox" id="' id '-stopped" name="stopped" checked')}}{{/check-input}}
+          {{#> check-label check-label--type="span"}}Stopped{{/check-label}}
+        {{/check}}
+        {{#> check  check--type="label" check--attribute=(concat 'for="' id '-down"') check--modifier="pf-c-select__menu-item"}}
+          {{#> check-input check-input--attribute=(concat 'type="checkbox" id="' id '-down" name="down" checked')}}{{/check-input}}
+          {{#> check-label check-label--type="span"}}Down{{/check-label}}
+        {{/check}}
+        {{#> check  check--type="label" check--attribute=(concat 'for="' id '-degraded"') check--modifier="pf-c-select__menu-item"}}
+          {{#> check-input check-input--attribute=(concat 'type="checkbox" id="' id '-degraded" name="degraded"')}}{{/check-input}}
+          {{#> check-label check-label--type="span"}}Degraded{{/check-label}}
+        {{/check}}
+        {{#> check  check--type="label" check--attribute=(concat 'for="' id '-needsMaintenance"') check--modifier="pf-c-select__menu-item"}}
+          {{#> check-input check-input--attribute=(concat 'type="checkbox" id="' id '-needsMaintenance" name="needsMaintenance" checked')}}{{/check-input}}
+          {{#> check-label check-label--type="span"}}Needs Maintenance{{/check-label}}
+        {{/check}}
+      </div>
+      <div class="pf-c-select__menu-group">
+        <div class="pf-c-select__menu-group-title">Vendor</div>
+        {{#> check  check--type="label" check--attribute=(concat 'for="' id '-dell"') check--modifier="pf-c-select__menu-item"}}
+          {{#> check-input check-input--attribute=(concat 'type="checkbox" id="' id '-dell" name="dell"')}}{{/check-input}}
+          {{#> check-label check-label--type="span"}}Dell{{/check-label}}
+        {{/check}}
+        {{#> check  check--type="label" check--attribute=(concat 'for="' id '-samsung"') check--modifier="pf-c-select__menu-item pf-m-disabled"}}
+          {{#> check-input check-input--attribute=(concat 'type="checkbox" id="' id '-samsung" name="samsung" aria-disabled="true" disabled')}}{{/check-input}}
+          {{#> check-label check-label--type="span"}}Samsung{{/check-label}}
```

Comment:
Are you saying just to add it to the label, or to move it from the check? If I remove it from check then that menu option still gets the hover.

Reply:
referencing the [check examples](https://pf-next.com/components/Check/examples/), looks like you need `disabled` on the `input` and `.pf-m-disabled` on whatever the label/text is. so you can just do

```suggestion
          {{#> check-label check-label--type="span" check-label--modifier="pf-m-disabled"}}Samsung{{/check-label}}
```

Reply:
I'm going to put it on both the label and the check because if it's not on the check, then you still get a hover highlight on that menu item, which is incorrect.